model/project.go

```diff
@@ -1542,6 +1551,25 @@ func (p *Project) IsGenerateTask(taskName string) bool {
 	return ok
 }
 
+func (p *Project) findAliasesForPatch(alias string, patchDoc *patch.Patch, catcher grip.Catcher) ([]ProjectAlias, error) {
```

Comment:
There's really no reason to pass the catcher into this function; it can operate independently and just return an error if it hits one, since we add that error to the catcher at 1459 anyway.